src/explore-education-statistics-frontend/src/middleware/pages/redirectPages.ts

```diff
@@ -0,0 +1,62 @@
+import redirectService, {
+  Redirects,
+  RedirectType,
+} from '@frontend/services/redirectService';
+import type { NextRequest } from 'next/server';
+import { NextResponse } from 'next/server';
+
+interface StoredRedirects {
+  redirects: Redirects;
+  fetchedAt: number;
+}
+
+let storedRedirects: StoredRedirects | undefined;
+
+const redirectPaths = {
+  methodologyRedirects: '/methodology',
+  publicationRedirects: '/find-statistics',
+};
+
+const cacheTime = 300000;
```

Comment:
I'm wondering if this delay, even if reduced, could cause problems in UI test runs. Can we make this easily changable or different for local + dev environments? It might make sense to always have it shorter on our local environments anyway.

Reply:
Good point, I've set it to zero for the test environment and added an optional param so that it can be overridden in the unit test to test the caching. It's only 1 min on other envs now so should be ok for dev, but we can always adjust it later if it causes problems.